internal/metrics/tripperware.go

```diff
@@ -0,0 +1,17 @@
+package metrics
+
+import (
+	"net/http"
+
+	"github.com/gol4ng/httpware"
+	"github.com/gol4ng/httpware/metrics"
+	"github.com/gol4ng/httpware/tripperware"
+)
+
+func Metrics(name string, recorder metrics.Recorder) httpware.Tripperware {
```

Comment:
maybe name it `Tripperware` instead of `Metrics` ?

Reply:
explain what this middleware does 👍

Reply:
its not a middleware

Reply:
lol, well explain why this `not a middleware` is great and useful 😛

Reply:
it s not a middleware because it's a tripperware